packages/babel-plugin-transform-flow-comments/README.md

```diff
@@ -1,10 +1,9 @@
 # babel-plugin-transform-flow-comments
 
-Turn flow type annotations into comments.
-
-You should be able to use this plugin instead of `babel-plugin-flow-strip-types` to preserve the `/* @flow */` directive and still use flow.
-
-http://flowtype.org/blog/2015/02/20/Flow-Comments.html
+> Turn flow type annotations into comments.
+>
+> You should be able to use this plugin instead of `babel-plugin-flow-strip-types` to preserve the `/* @flow */` directive and still use flow.
+> [Flow Comments Blog Post](http://flowtype.org/blog/2015/02/20/Flow-Comments.html)
```

Comment:
On the website, package descriptions are removed. They need to be on one single line.

Could you maybe split the comment into two parts ? Thanks.